pkg/spec/spec.go

```diff
@@ -71,6 +71,7 @@ type SecretMod struct {
 type App struct {
 	Name                       string            `json:"name"`
 	Labels                     map[string]string `json:"labels,omitempty"`
+	Controller                 string            `json:"controller,omitempty"`
```

Comment:
@surajssd I'm fine with this, can you add a comment on top of this that this is a no-op field and is only being used for OpenAPI generation, and should not be used in the code elsewhere. Use spec.Controller.Controller instead.
Something like that. This is good to go after that!

Reply:
I am not sure about the comments because this will show up in the generated openAPI specification. But for now I have added the comment.